HTTPServerProject.Tests/HTTPServerProxyTests/ResponseTests.cs

```diff
@@ -0,0 +1,224 @@
+using HTTPServerReadTests.Streams;
+using HTTPServerProxy.Response;
+using HTTPServerWriteTests.Streams;
+
+namespace HTTPServerProxyTests.Response;
+
+public class ProxyResponseTests
+{
+    [Fact]
+    public void GetTodoPOSTValidResponseTest()
+    {
+        var expectedStatusLine = "HTTP/1.1 201 Created";
+        var proxyStream = new List<string>() 
+		{
+        "HTTP/1.1 200 OK", 
+		"Content-Type: application/json", 
+		"Content-Length: 20", 
+		"", 
+		"{ 'task': 'First Item' }" 
+		};
+        var stream = new List<string>();
+        var reader = new TestReadStreams(proxyStream);
+        var httpType = "POST";
+        var httpPath = "todo";
+        var writer = new TestWriteStreams(stream);
+
+        var proxyResponse = new ProxyResponse(reader: reader, writer: writer, path: httpPath, type: httpType);
+        proxyResponse.GetResponse();
+        var actualStatusLine = stream[0];
+        Assert.Equal(expectedStatusLine, actualStatusLine);
+    }
+
+    [Fact]
+    public void GetTodoPOSTValidResponseNoBodyTest()
+    {
+        var expectedStatusLine = "HTTP/1.1 201 Created";
+        var proxyStream = new List<string>() 
+		{ 
+			"HTTP/1.1 200 OK", 
+			"Content-Type: application/json", 
+			"Content-Length: 20", 
+			"", 
+			"" 
+		};
+        var stream = new List<string>();
+        var reader = new TestReadStreams(proxyStream);
+        var httpType = "POST";
+        var httpPath = "todo";
+        var writer = new TestWriteStreams(stream);
+
+        var proxyResponse = new ProxyResponse(reader: reader, writer: writer, path: httpPath, type: httpType);
+        proxyResponse.GetResponse();
```

Comment:
The name `GetResponse` doesn't make sense. It appears to write to the write stream, it doesn't return a response. Are you trying to use the command pattern here? (quick refresher) https://www.tutorialspoint.com/design_pattern/command_pattern.htm

Reply:
Yes, I was trying to do something like that. I can rename the `GetResponse` method to `Execute`